edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CrdWatcher.cs

```diff
@@ -204,9 +213,10 @@ private async Task HandleEdgeDeploymentDeleted(V1ServiceList currentServices, V1
             this.currentModules = ModuleSet.Empty;
         }
 
-        private async Task UpsertDeployments(V1ServiceList currentServices, V1DeploymentList currentDeployments, EdgeDeploymentDefinition<TConfig> customObject)
+        private async Task UpsertDeployments(V1ServiceList currentServices, V1DeploymentList currentDeployments, IList<KubernetesModule<TConfig>> spec)
         {
-            var desiredModules = ModuleSet.Create(customObject.Spec.ToArray());
+            var desiredModules = ModuleSet.Create(spec.ToArray());
+            Console.WriteLine($"DESIRED MODULES:\n {desiredModules}");
```

Comment:
Shouldn't we delete all Console.WriteLine calls?

Reply:
Yes, it should - missed that one.